core/src/components/title/title.ios.scss

```diff
@@ -40,16 +40,17 @@
 // --------------------------------------------------
 
 :host(.title-large) {
-  @include padding(0, 16px);
+  @include padding(2px, 12px, 4px, 12px);
   @include transform-origin(left, center);
 
-  bottom: 0;
+  position: static;
 
   align-items: flex-end;
 
   min-width: 100%;
 
-  font-size: 34px;
+  font-size: dynamic-font-max(34px, 1.8);
```

Comment:
![Screenshot 2023-08-29 at 4 50 28 PM](https://github.com/ionic-team/ionic-framework/assets/13530427/dbbcba6b-a97d-4b83-8c4b-9c2c4cd6a36f)

I used the Contacts app as a reference.